doc/how_to_use.md

```diff
@@ -0,0 +1,131 @@
+# How to use Sōzu
+
+> Before to deep dive in the configuration part of the proxy. You should take a look at the [getting start](./getting_started.md) if you haven't done it yet.
+
+## Configuration file
+
+> The configuration file is a [.toml](https://github.com/toml-lang/toml)
+
+Sōzu configuration process involves 3 major sources of parameters:
+
+* The `global` section, which sets process-wide parameters.
+* The definition of the frontends like `https`, `http`, `tcp`.
+* The proxies sections under: `[applications]`.
+
+### Global parameters
+
+Parameters in the global section can be use to define the global settings shared by the master and workers (like the log level):
+
+* `command_socket` path to the unix socket command (see below for more infos)
+* `saved_state` path to file to dump the proxy state
```

Comment:
this is not the path where the proxy dumps the state, it's the path from which it tries to load its state at startup